### ISSUE TITLE
Markers created around assert statements

### ISSUE THREAD

**Issue by SvenjaScherrer** [DeadCodeProductions/program-markers#52]
When instrumenting the following program:
```
#include "csmith.h"
g_101() {
    int32_t l_981 = 2;
    if (0) {
        assert (g_101 == &l_981);
    }
}
```
This is the resulting instrumented program:
```
void VRMarker0_(void);
#define VRMARKERMACRO0_(VAR, TYPE) if (!(((VAR) >= 0) && ((VAR) <= 0))) { VRMarker0_(); }
#include "csmith.h"
g_101() {
    int32_t l_981 = 2;
    VRMARKERMACRO0_(assert (g_101 == &l_981),"int32_t")
if (0) {
        assert (g_101 == &l_981);
    }
}
```
The instrumented program does not compile (error: invalid operands to binary expression ('void' and 'int')).

**Comment by thetheodor** [comment]
The other options it first preprocess the code  (e.g., `clang -E  -P -I/path/to/csmith_inc_dir test.c > testpp.c`) and work on the preprocessed file

**Comment by thetheodor** [comment]
Fixed with #54
